src/app/workflow/versions/versions.component.html

```diff
@@ -217,7 +217,18 @@
         >
       </th>
       <td mat-cell *matCellDef="let version">
-        <mat-icon data-cy="metrics" *ngIf="version.metricsByPlatform && (version.metricsByPlatform | json) !== '{}'">check</mat-icon>
+        <img
+          *ngIf="hasExecutionMetrics"
+          src="assets/svg/manufacturing.svg"
```

Comment:
I think you need a relative path here, `../../assets`, or however many double dots are necessary, or it won't work in prod. Here's where it matters: https://github.com/dockstore/dockstore-ui2/blob/develop/.circleci/config.yml#L247